torch/__init__.py

```diff
@@ -2077,3 +2077,27 @@ def _constrain_as_size(symbol, min: Optional[builtins.int] = None, max: Optional
 
 from . import _logging
 _logging._init_logs()
+
+
+def import_device_backends():
+    if sys.version_info < (3, 10):
+        from importlib_metadata import entry_points
+    else:
+        from importlib.metadata import entry_points
+
+    for backend in entry_points(group='torch.backends'):
+        try:
+            # just load the plugin without calling
+            backend.load()
+        except Exception:
```

Comment:
I think it would be better to show failure information when an exception is encountered.